### DISCUSSION TITLE
filtering criteria scATAC-seq, snATAC-seq, multiome kits

### DISCUSSION THREAD

**Discussion by tanasa** [stuart-lab/signac#1117] [Q&A] [answered]
Hi everyone, 

Would you please let me know if we shall use modified filtering criteria when we work with : 

<> scATAC-seq

<> snATAC-seq

<> multiome kits (scATACseq- + scRNA-seq)

Thanks a lot, 

Bogdan

**Comment by timoast** [comment, marked as answer]
You should use modified filtering criteria, but I can't tell you specific numbers because they will vary for each individual dataset. You will need to look at the distribution of QC values for your dataset and make a decision that you think makes sense.
